pkgs/applications/virtualization/qemu/default.nix

```diff
@@ -116,7 +116,12 @@ stdenv.mkDerivation rec {
       sha256 = "sha256-evzrN3i4ntc/AFG0C0rezQpQbWcnx74nXO+5DLErX8o=";
     })
   ]
-    ++ lib.optional nixosTestRunner ./force-uid0-on-9p.patch;
+  ++ lib.optional nixosTestRunner ./force-uid0-on-9p.patch
+  # fix 9p on macOS host, landed in master
+  ++ lib.optional stdenv.isDarwin (fetchpatch {
```

Comment:
Please make this unconditional

Reply:
It's better to avoid conditional patches even when it doesn't affect other systems, because it means the maintainer has less combinations to test and is more likely to catch broken patches for other systems.